src/pivotal-ui/components/header/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "homepage": "http://styleguide.pivotal.io/objects.html#header",
   "dependencies": {
-    "@npmcorp/pui-css-drop-down-menu": "^1.3.4-0",
+    "@npmcorp/pui-css-drop-down-menu": "^1.3.4",
     "@npmcorp/pui-css-typography": "^4.0.0"
   },
-  "version": "3.0.0"
+  "version": "3.0.1-1"
```

Comment:
Not that it really matters, but should this have been `3.0.1-alpha.1` instead? If you used a tool to automatically bump this version, then please ignore this.